frontend/src/Plugin.js

```diff
@@ -143,6 +144,8 @@ class ShopgateAnalytics extends SgTrackingPlugin {
         sdkData.meta = rawData.meta;
       }
 
+      const checkoutCompletedEvent = new HttpRequest();
```

Comment:
Also the url is missing for the endpoint

Reply:
Updated with the url

Reply:
Modified used url to account for development or production